src/material/autocomplete/testing/autocomplete-harness.ts

```diff
@@ -66,15 +66,19 @@ export class MatAutocompleteHarness extends ComponentHarness {
   }
 
   /** Gets the options inside the autocomplete panel. */
-  async getOptions(filters: OptionHarnessFilters = {}): Promise<MatAutocompleteOptionHarness[]> {
-    return this._documentRootLocator.locatorForAll(MatAutocompleteOptionHarness.with(filters))();
+  async getOptions(filters: OptionHarnessFilters = {}): Promise<MatOptionHarness[]> {
```

Comment:
can you use the `Omit` type for the param to remove the `ancestor` property since we just overwrite it anyways